### ISSUE TITLE
resource.distribution.contact.identifier.present 

### ISSUE THREAD

**Issue by gothub** [NCEAS/metadig-checks#55]
## Description

Check if a distribution contact identifier has been provided, such as an ORCID

## Priority

FAIR: Optional

## Issues
- EML does not provide this item
- I have yet to see this item in DataONE metadata for any of the MNs that use ISO: NCEA, PANGAEA

## Procedure

- check if the distribution contact id is present, and if not, the check fails.

**Comment by JEDamerow** [comment]
@gothub @mbjones  Is there a rationale for this check to have Accessible label? The Contact present check #122 has Reusable label, which made more sense to me.

**Comment by gothub** [comment]
@JEDamerow This item is the identifier (ORCID, URL?) for the organization or individual that is responsible for providing access to data, i.e. for a data archive, NASA DAAC, etc, so that is why it's in the `Accessible` category.

**Comment by jeanetteclark** [comment]
@gothub it looks like this does not have any EML paths? 

I think the relevant one here would be `eml/*/contact/userId` (related to issue #54 )

**Comment by gothub** [comment]
@jEDamerow @jeanetteclark The distributor contact is the person or entity that is specifically responsible for enabling access to data. Have a look at https://geo-ide.noaa.gov/wiki/index.php?title=ISO_Distribution_Information for ISO. 

@mbjones does the EML path that @jeanetteclark indicate cover this concept for EML?

**Comment by mbjones** [comment]
@gothub yes, the path looks fine.  In EML, the `/eml/dataset/contact` field will be the person to contact for enabling access, and it contains a `userId` child element with the identifier for that contact.

**Comment by gothub** [comment]
@mbjones @jeanetteclark should we also be checking:
```
/eml/*/associatedParty/role[RoleType='distributor']
```

**Comment by jeanetteclark** [comment]
It doesn't hurt, but I almost never see EML in the wild that implements that.

**Comment by gothub** [comment]
It's from the EDI EML best practices v3, so I'm assuming that they use it.

**Comment by mbjones** [comment]
I guess we could, but `contact` is really the place for this info in EML.  So, if it is missing from contact then I think the test should probably fail. Maybe something to discuss with folks from EDI, as I am sure they are also using the `contact` field.
